HACKING.md

```diff
@@ -0,0 +1,90 @@
+# Hacking on Libreport
+
+Here's where to get the code:
+
+    $ git clone https://github.com/abrt/libreport.git
+    $ cd libreport/
+
+The remainder of the commands assume you're in the top level of the
+Libreport git repository checkout.
+
+## Building
+
+1. Install dependencies
+
+    First you should install all dependent packages.
+
+    Dependencies can be listed by:
+
+        $ ./autogen.sh sysdeps
+
+    or installed by:
+
+        $ ./autogen.sh sysdeps --install
+
+    The dependency installer gets the data from [the rpm spec file](libreport.spec.in)
+
+2. Build from source
+
+    When you have all dependencies installed you can now build a rpm package by these commands:
+
+        $ ./autogen.sh
+        $ make rpm
+
+    Now in the `build` folder you can find a rpm packages both in `noarch` and `x86_64` (or your
+            architecture) directories.
+
+
+### Testing
+
+Easiest way how to test everything (build, run tests) is to make rpm build (see,
+        *2. Build from source in Building chapter*).
```

Comment:
Actually the original is also correct (https://en.wikipedia.org/wiki/Zero-marking_in_English). “A source” would be confusing, since it doesn’t refer to anything specific.